lms/tasks/email_digests.py

```diff
@@ -146,27 +142,10 @@ def send_instructor_email_digest(
     # Caution: datetime.fromisoformat() doesn't support all ISO 8601 strings!
     # This only works for the subset of ISO 8601 produced by datetime.isoformat().
     created_before = datetime.fromisoformat(created_before)
-
-    if created_after is None:
-        created_after = created_before - timedelta(days=7)
-    else:
-        created_after = datetime.fromisoformat(created_after)
+    created_after = datetime.fromisoformat(created_after)
 
     with app.request_context() as request:  # pylint:disable=no-member
         with request.tm:
-            task_done_data = _get_task_done_data(request.db, h_userid)
-
-            if task_done_data:
-                created_after = max(
-                    datetime.fromisoformat(task_done_data["created_before"]).replace(
-                        tzinfo=timezone.utc
-                    ),
-                    created_after.replace(tzinfo=timezone.utc),
-                    # Don't count annotations from before we deployed https://github.com/hypothesis/lms/pull/5904.
-                    # This line can safely be removed on Weds 20th Dec 2023 or later.
-                    datetime(year=2023, month=12, day=13, hour=5, tzinfo=timezone.utc),
-                )
```

Comment:
No more potentially overriding `created_after` with data from `task_done`. Actually, due to the `max()`, it probably isn't necessary to remove this: this block of code can only shorten the `created_before - created_after` time range not widen it. (And it should never do that either: there shouldn't ever be a `created_before` in `task_done` that's newer than the now-fixed `created_after`). This code is not necessary right now so let's just remove it from the equation for good measure, simplifies things.